components/home/menu/calendar/Screens/DashboardScreen/index.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, Alert } from 'react-native';
+import { Agenda } from 'react-native-calendars';
+import styles from './styles';
+
+export default class DashboardScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: {},
+      synchronizedEvents: this.structureSynchronizedEvents(
+        props.navigation.state.params.jsonFile.items
+      )
+    };
+  }
+
+  structureSynchronizedEvents(events) {
+    let tempArray = [];
+    events.forEach(event => {
+      tempArray.push(
+        {
+          date: event.start.dateTime != null ? event.start.dateTime.substring(0,event.start.dateTime.indexOf('T')):event.start.date,
+          title: event.summary!=null ? event.summary:'No Title For this Event',
+          startTime: event.start.dateTime != null ? event.start.dateTime : event.start.date,
+          endTime: event.end.dateTime != null ? event.end.dateTime : event.end.date,
+          description: event.description != null ?event.description : '',
+          address: ''
+        }
+      )
+    });
+    this.setState({
+      synchronizedEvents: this.tempArray
+    });
+    console.log('temp:\n', tempArray, '--->\n\n\-n>>>>');
+    return tempArray;
+  }
+
+  loadItems(day) {
+    setTimeout(() => {
+      for (let i = -15; i < 85; i++) {
+        const time = day.timestamp + i * 24 * 60 * 60 * 1000;
+        const strTime = this.timeToString(time);
```

Comment:
Use constants to store time conversions. Also you can just do
strTime = this.timeToString(day.timestamp + i * 24 * 60 * 60 * 1000)

Reply:
Sorry, but I don't see the problem. Can you be more specific please.